lib/internal/histogram.js

```diff
@@ -184,9 +182,7 @@ class Histogram {
   percentile(percentile) {
     if (!isHistogram(this))
       throw new ERR_INVALID_THIS('Histogram');
-    validateNumber(percentile, 'percentile');
-    if (NumberIsNaN(percentile) || percentile <= 0 || percentile > 100)
-      throw new ERR_OUT_OF_RANGE('percentile', '> 0 && <= 100', percentile);
+    validateNumber(percentile, 'percentile', 1, 100);
```

Comment:
I tried this approach before, but all the numbers between 0 and 1 are not allowed any more.
This(https://github.com/nodejs/node/pull/45772#discussion_r1042514857) is previous discussion.

Reply:
@deokjinkim Thank you for explaining the history. I'll close it.